algorithm/soil/soil_addincrement.yaml.j2

```diff
@@ -0,0 +1,74 @@
+state geometry:
+  fms initialization:
+    namelist filename: ./fv3jedi/fmsmpp.nml
+    field table filename: ./fv3jedi/field_table
+  akbk: ./fv3jedi/akbk.nc4
+  layout:
+  - {{ soil_layout_x }}
+  - {{ soil_layout_y }}
+  npx: {{ soil_npx_ges }}
+  npy: {{ soil_npy_ges }}
+  npz: {{ soil_npz_ges }}
+#  field metadata override: ./fv3jedi/fv3jedi_fieldmetadata_history.yaml
+increment geometry:
+  fms initialization:
+    namelist filename: ./fv3jedi/fmsmpp.nml
+    field table filename: ./fv3jedi/field_table
+  akbk: ./fv3jedi/akbk.nc4
+  layout:
+  - {{ soil_layout_x }}
+  - {{ soil_layout_y }}
+  npx: {{ soil_npx_anl }}
+  npy: {{ soil_npy_anl }}
+  npz: {{ soil_npz_anl }}
+#  field metadata override: ./fv3jedi/fv3jedi_fieldmetadata_fv3inc.yaml
+state:
+  state variables: 
+    - soilMoistureVolumetric
+    - stc
+    - totalSnowDepth
+    - sheleg
```

Comment:
Do we need keep `totalSnowDepth` and `sheleg` here in soil DA? Suggest to remove them